source/features/default-to-rich-diff.tsx

```diff
@@ -0,0 +1,40 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+const observer = new MutationObserver(([{addedNodes}]) => {
+	for (const node of addedNodes) {
+		if (node instanceof Element) {
+			if (node.matches('.js-diff-progressive-container')) {
+				observer.observe(node, {childList: true});
+			} else {
+				setRichDiff(node);
+			}
+		}
+	}
+});
+
+function setRichDiff(node: Element | null) {
+	if (node) {
+		for (const fileHeader of select.all('.file-header[data-file-type=".svg"] [aria-label="Display the rich diff"]', node)) {
+			fileHeader.click();
+		}
+	}
+}
+
+function init() {
+	setRichDiff(select('.js-diff-progressive-container:first-child'));
```

Comment:
You could move the `.js-diff-progressive-container:first-child` to the `select.all` call in `setRichDiff`.